.k8s_ci.Dockerfile

```diff
@@ -21,7 +21,7 @@ ARG SERVER_DIR=$ROOT_DIR/server
 
 # For much faster build time override this with image0 (Dockerfile.0 build):
 #   docker build --build-arg BASE=<image0 name>...
-ARG STAGE1_BASE=ubuntu:20.04
+ARG STAGE1_BASE=python:3.7-slim
```

Comment:
I think we need additional testing of downstream effects before using this as the base image. The main reason is that the galaxy container is used as the default/fallback container in the absence of a mulled container for a tool, as well as for tools that require LIB_GALAXY. The Ubuntu image was chosen because it provided a very small image and a reasonable toolset. If this base debian image doesn't take too much of that away, the size reduction is quite enticing. Ping @almahmoud @afgane

Reply:
I checked /usr/bin and /bin for both images and compared using meld.

Present in ubuntu:20.04 but not python:3.7-slim:
- All bzip2 utilities
- free
- kill
- locale-check
- man
- pgrep
- pkill
- pmap
- ps
- pwdx
- select-editor
- sensible-browser
- sensible-editor
- sensible-pager
- skill
- slabtop
- snice
- tload
- uptime
- vmstat
- w
- w.procps
- watch

Present in python:3.7-slim
- Python binaries obviously, but we install those on the ubuntu image.
- openssl

## Package differences:
Ubuntu has:
- bzip2
- libcrypt1
- libprocps
- logsave
- lsb-base
- procps
- sensible-utils

So ubuntu has more process management stuff. But not really more tools (except bzip2). I don't think bzip2 is essential. If we do not want to take that risk, it can be installed on the image.

Reply:
Thanks for this fantastic comparison! I suspect that some of these scripts might use is `kill`. bzip2 too is possibly another contender, but let's see what the tool tests reveal.